app/controllers/RequestDataController.scala

```diff
@@ -44,8 +44,13 @@ class RequestDataController @Inject() (
 
   def onPageLoad: Action[AnyContent] = (identify andThen profileAuth andThen getData andThen requireData).async {
     implicit request =>
-      downloadDataConnector.getEmail(request.eori).map { email =>
-        Ok(view(email.address))
+      downloadDataConnector.getEmail(request.eori).map {
```

Comment:
same here

Reply:
Referencing the test, this is what logErrorsAndContinue does already :)

```
val redirectUrl = Some(RedirectUrl(routes.IndexController.onPageLoad.url))

        status(result) mustEqual SEE_OTHER
        redirectLocation(result).value mustEqual routes.JourneyRecoveryController.onPageLoad(redirectUrl).url
```